src/coreComponents/constitutive/fluid/multifluid/CO2Brine/CO2BrineFluid.cpp

```diff
@@ -157,7 +157,8 @@ void CO2BrineFluid< PHASE1, PHASE2, FLASH >::checkTablesParameters( real64 const
     m_phase1->enthalpy.checkTablesParameters( pressure, temperatureInCelsius );
   } catch( SimulationError const & ex )
   {
-    string const errorMsg = GEOS_FMT( "{}: Table input error for phase no. 1.\n", getDataContext() );
+    string const errorMsg = GEOS_FMT( "Table input error for {} phase (in table from \"{}\").\n",
+                                      m_componentNames[m_p1Index], m_phasePVTParaFiles[m_p1Index] );
```

Comment:
it should be `m_phaseNames` i think

Reply:
@paveltomin I used `m_componentNames` because of the preferences of the users that I have surveyed. Why should it be `m_phaseNames` for you?

Reply:
Because it says "phase" in the message
If you use component name, it should say "component"

Reply:
Oh, alright! I'm changing that